src/tests/ftest/util/dmg_utils.py

```diff
@@ -236,15 +236,16 @@ def storage_reset(hosts, user=None, hugepages="4096", insecure=True):
     return result
 
 
-def pool_create(path, host_port, scm_size, insecure=True, group=None,
+def pool_create(path, scm_size, host_port=None, insecure=True, group=None,
```

Comment:
(pylint-too-many-arguments) Too many arguments (11/10)

Reply:
There's no easy way to reduce the number of parameters for this method.